Food_Tracker_App/app/src/main/java/com/example/foodtracker/IngredientAdapter.java

```diff
@@ -0,0 +1,60 @@
+package com.example.foodtracker;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.ArrayList;
+
+//TO DO: will have to "expand" the ingredients list
+public class IngredientAdapter extends ArrayAdapter<Ingredient> {
```

Comment:
If we want to expand the ingredients list should we use an `ExpandableArrayAdapter` instead?

Reply:
sure, but I'll probably need someone to help me with implementing that